util/configure

```diff
@@ -359,6 +368,12 @@ for my $opt (@ARGV) {
     }
 }
 
+for my $lib (qw/openssl pcre zlib/) {
+    if ($with_ext_lib{$lib} && !$with_ext_lib_opts{"$lib-opt"}) {
```

Comment:
Ditto, let's save the trouble of constructing the key `"$lib-opt"`. Just use `$lib` instead.